onnxruntime/test/contrib_ops/decoder_masked_multihead_attention_op_test.cc

```diff
@@ -766,127 +769,134 @@ TEST(DecoderMaskedSelfAttentionTest, Test_fp16) {
     return;
   }
 
-  // Vary batch size
-  for (int batch_size = 1; batch_size <= 5; batch_size += 2) {
-    // Vary kv_lengths
-    for (int past_sequence_length = 1; past_sequence_length <= 3000; past_sequence_length += 150) {
-      int sequence_length = 1;
-      int number_of_heads = 12;
+  struct MyTestCase {
+    int batch_size;
+    int past_sequence_length;
+    int hidden_size;
+  } test_cases[] = {
+      {1, 1, 768},
```

Comment:
Can we add a few more test combinations (same request for fp32 as well)? It seems like past_sequence_length cases are dramatically reduced. Now that it only takes 15 sec, maybe we can add 10 more cases (it will still be lower than baseline) ?

Reply:
Sure. Let me add more cases.